engine/src/dpx_cidtype0.rs

```diff
@@ -2667,11 +2662,7 @@ pub unsafe extern "C" fn CIDFont_type0_t1cdofont(mut font: *mut CIDFont) {
     free(CIDToGIDMap as *mut libc::c_void);
     CIDFontInfo_close(&mut info);
     if verbose > 1i32 {
-        dpx_message(
-            b"[%u glyphs][%d bytes]\x00" as *const u8 as *const i8,
-            num_glyphs as i32,
-            destlen,
-        );
+        info!("[{} glyphs][{} bytes]", num_glyphs as i32, destlen);
```

Comment:
Program behavior became a tiny bit different after this change: vprintf and friends will type pun the i32 as u32 when it sees the "%u" format. The rust `format_args` macro doesn't do this.

Reply:
I just deleted `as i32` in separate commit because variable have type `u16`